woodwork/type_sys/inference_functions.py

```diff
@@ -52,6 +53,9 @@ def integer_nullable_func(series):
             return not _is_categorical_series(series, threshold)
         else:
             return True
+    elif pdtypes.is_float_dtype(series.dtype):
```

Comment:
How do we handle specificity. So if one column were to parse 80% successful as integer and 90% successful as float, which would we pick?

Reply:
Yep good question, here's how the logic is determined for which type is picked in cases of multiple matches: https://woodwork.alteryx.com/en/latest/guides/custom_types_and_type_inference.html#Logical-Type-Relationships